lib/dock.rb

```diff
@@ -0,0 +1,46 @@
+class Dock
+
+  attr_reader :name,
+              :max_rental_time,
+              :boat_inventory,
+              :total_renters,
+              :total_rental_by_person
+  def initialize(name, max_rental_time)
+    @name = name
+    @max_rental_time = max_rental_time
+    @boat_inventory = {}
+    @total_renters = []
+    @log_hour = 0
+    @total_rental_by_person = {}
+  end
+
+  def add_boats_to_inventory(type)
+    @boat_inventory[type] = type.price_per_hour
+  end
+
+  def log_hour
```

Comment:
The log hour method should increase the number of hours logged on each `Boat`.  I think it would make sense to have some sort of collection that tracked rentals (maybe a hash with renter as the key and an array of boats as the value).  Then you would iterate over those boats and add an hour to each boat.